prac_08/silver_service_taxi_test.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP1404/CP5632 Practical
+Silver Service Taxi Test
+"""
+
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+
+    hummer = SilverServiceTaxi("Hummer", 100, 2)
+    hummer.drive(18)
+    print(hummer)
+    print(hummer.get_fare())
```

Comment:
How do you know if this worked correctly? 
Your tests should have something in them to show whether they pass or fail - like we did with Guitar expected/actual.